README.md

```diff
@@ -1,3 +1,48 @@
+
+
+
+
 # Jernl
 
-It's journaling, with a calendar. Sort of based on the old Seinfeld yarn re: keeping the chain going, partly a solution to not wanting to pay for Notion. Signup is free: [jernl.space/register](https://jernl.space/register).
\ No newline at end of file
+## Installation
+
+To install all of the template files, run the following script from the root of your project's directory:
+
+```
+bash -c "$(curl -s https://raw.githubusercontent.com/CFPB/development/main/open-source-template.sh)"
+```
+
+----
+
+# Project Title
+
+**Description**:  It's journaling, with a calendar. Sort of based on the old Seinfeld yarn re: keeping the chain going, partly a solution to not wanting to pay for Notion.
+
+Other things to include:
+
+  - **Technology stack**: Laravel 7
+  - **Status**:  Beta
+  - **Links to production or demo instances**: https://jernl.charlesharri.es/
+
+
+**Screenshot**:
+
+![](https://raw.githubusercontent.com/charlesharries/jernl/main/screenshot.jpg)
```

Comment:
This screenshot doesn't exist—just returns a 404. Screenshot would probably be a nice feature for the readme though, I'll make a note to add one proper.

Reply:
Oh yeah of course, yeah that should show up once it's merged, didn't spot that screenshot.jpg at the bottom of the review 🤦‍♂️